server/stream.go

```diff
@@ -901,7 +915,7 @@ func getExpectedLastSeq(hdr []byte) uint64 {
 }
 
 // processInboundJetStreamMsg handles processing messages bound for a stream.
-func (mset *Stream) processInboundJetStreamMsg(_ *subscription, pc *client, subject, reply string, msg []byte) {
+func (mset *Stream) processInboundJetStreamMsg(sub *subscription, pc *client, subject, reply string, msg []byte) {
```

Comment:
I see you set sub but I don't think you use it.

Reply:
Left over from some earlier code, will remove.